plugins/dev-tools/README.md

```diff
@@ -47,5 +47,24 @@ addGUIControls((options) => {
 }, defaultOptions);
 ```
 
+### Populate Random
+
+The `populateRandom` function adds random blocks to a workspace. Blocks are selected from the full set of defined blocks. Pass in a worskpace and how many blocks should be created.
+```js
+import {populateRandom} from '@blockly/dev-tools';
+// Add 10 random blocks to the workspace.
+populateRandom(workspace, 10);
+```
+
+The `populateRandomButton` function adds a button to the page with `populateRandom` as its onClick function.
```

Comment:
This is okay for now, but just keep in mind this will become redundant once the playground has support for adding buttons.